conf/messages.en

```diff
@@ -349,5 +349,32 @@ confirmationPage.printsave.link = Print or save a declared copy of the estate’
 confirmationPage.subheading1 = What happens next
 confirmationPage.paragraph1 = Keep a note of your reference in case you need to contact HMRC. If there is a problem with the declaration, we will contact {0}.
 confirmationPage.personalRepresentative.default = the personal representative
+confirmationPage.paragraph2 = Your request to close this estate will be processed and access to its online register will be removed.
+confirmationPage.paragraph3 = If this has been done in error, you can
+confirmationPage.paragraph3.link = contact the Deceased Estates helpline to reopen the records (opens in a new tab)
```

Comment:
Everywhere else in messages the content is `contact the deceased estate helpline`